sdk/src/wallet/migration/migrate_0.rs

```diff
@@ -11,6 +11,77 @@ use crate::wallet::Error;
 
 pub struct Migrate;
 
+fn migrate_native_token(output: &mut serde_json::Value) {
```

Comment:
This may be a problem in the case where we are migrating an older storage. Part of the purpose of the `Convert` trait is checking whether we already migrated.

```rust
 if serde_json::from_value::<Self::New>(value.clone()).is_err()
```

If we don't do this it's conceivable that we will have already migrated, but since we don't have the version in storage we try to do it again. Your code would panic then.

Reply:
Ah, nevermind. I thought the structure was changing but it's just a rename.